src/desktop/lib/global_client_setup.tsx

```diff
@@ -9,9 +9,6 @@ import { hydrate as hydrateStitch } from "@artsy/stitch/dist/internal/hydrate"
 import { initModalManager } from "desktop/apps/authentication/client/index"
 import { Components } from "@artsy/stitch/dist/internal/types"
 
-// Mount global router
-import "desktop/apps/experimental-app-shell/client"
```

Comment:
I remember this being deliberate, but I'm not sure exactly why right now :o

Reply:
Related to https://github.com/artsy/force/pull/4851/files#diff-38aabbe53c49d88c4dfa45f4f44309e4R2

Reply:
Hmm, I didn't notice anything going on? This also seems odd to have in here, unless I'm misunderstanding.

Basically, that file contains all client-side code needed when loading one of the Reaction routes. So by keeping it in that separate asset file currently, we do accomplish that.

It does work placing it here (but is redundant with that asset file which can thus be removed?). However, it won't be needed and won't do anything on non Reaction pages.

Do you think it's better to have it here, and have that separate asset file removed? I'm not sure which is better.

Reply:
The thinking is that this particular asset is global, and will cover multiple pages, versus the specific asset being loaded by the jade when navigating to a particular route. Since this crosses many routes it needs to be hoisted up (I think).

Reply:
Right, this covers multiple pages, but the [same layout](https://github.com/artsy/force/blob/0d50ace258d7b94b8a7fd430da5376295156bf2d/src/desktop/components/main_layout/templates/react_blank_index.jade) is used for all of the specific routes and will [load](https://github.com/artsy/force/blob/0d50ace258d7b94b8a7fd430da5376295156bf2d/src/desktop/components/main_layout/templates/blank.jade#L29) the corresponding assets.

And then that won't load on the other routes, which by definition don't need this code.

So that's why it seems perfect to remove it from the global include here, and keep it in that specific asset file, and indeed everything seemed to work fine.

Maybe let's leave it out of here for now, and re-add if something occurs?